monai/visualize/img2tensorboard.py

```diff
@@ -14,20 +14,26 @@
 import PIL
 from PIL.GifImagePlugin import Image as GifImage
 from torch.utils.tensorboard import SummaryWriter
+from torch import Tensor
 from tensorboard.compat.proto import summary_pb2
 from monai.transforms.utils import rescale_array
 
+from typing import Sequence, Union
 
-def _image3_animated_gif(imp, scale_factor=1):
+ndarray = np.array
```

Comment:
You used both `ndarray` and `np.array` in this file.
Suggest using `np.array` directly.